docs/source/user-manual/copy-paste-blocks.md

```diff
@@ -0,0 +1,43 @@
+---
+myst:
+  html_meta:
+    "description": "Copy, cut and paste blocks in Volto, the Plone 6 frontend."
+    "property=og:description": "Copy, cut and paste blocks in Volto, the Plone 6 frontend."
+    "property=og:title": "How to copy, cut, and paste Volto blocks"
+    "keywords": "Volto, Plone, frontend, React, User manual, edit blocks, copy, cut, paste"
+---
+
+(copy-cut-paste-blocks-label)=
+
+# Copy, Cut, and Paste blocks in Volto
+
+Volto supports copying/cutting of blocks from one page and pasting them to another page. Here's how you can do:
+
+(copy-cut-blocks-label)=
+
+## Copy/Cut blocks
+
+While in edit mode you can hold <kbd>ctrl</kbd> key to select multiple blocks you want to copy/cut.
```

Comment:
Avoid inserting HTML into Markdown because it is not supported in PDF and other output formats. Use MyST markup instead.

```suggestion
While in edit mode you can hold :kbd:`ctrl` key to select multiple blocks you want to copy/cut.
```

Reply:
Oh... Thanks Steve! didn't knew that. Noted for next time :)